src/components/GamePage.vue

```diff
@@ -71,7 +71,13 @@
       },
       submit: function () {
         console.log(`submitting text for ${this.sharedState.playerId}`)
-        this.suggestionDisabled = 1
+        this.suggestionDisabled = true
+        request.post('/api/submit', {
+          playerId: this.sharedState.playerId,
+          suggestionDisabled: this.suggestionDisabled
+        }, function (err, state) {
+          if (err) { console.log(err) }
```

Comment:
This hides the error, should either be assert(!err) or window.alert(err)

(We should unify all error handling/display code into one place)